test262/main.js

```diff
@@ -229,6 +245,52 @@
             byteCodePerformancePerTestResults.total,
         });
       }
+
+      // chart-test262-bytecode-performance
+      const optimizedByteCodePerformanceTests =
```

Comment:
We call it `bytecode-optimized` elsewhere, so `byteCodeOptimized...` pls :)

Also, the comment above doesn't match

Reply:
updated